f5_openstack_agent/lbaasv2/drivers/bigip/resource_manager.py

```diff
@@ -499,6 +519,9 @@ def _create(self, bigip, vs, listener, service):
         super(ListenerManager, self)._create(bigip, vs, listener, service)
 
     def _update(self, bigip, vs, old_listener, listener, service):
+        tls = self.driver.service_adapter.get_tls(service)
+        if tls:
+            self._create_ssl_profile(bigip, vs, tls)
```

Comment:
If the user manually modify some settings in client ssl profile, will your code keep those unchanged and only modify the certificates?

Reply:
And if the VS also has some other profiles manually inserted by the user, will those profiles still attach to the VS after updating certificate?

Reply:
> If the user manually modify some settings in client ssl profile, will your code keep those unchanged and only modify the certificates?

Yes, this  code will  keep the manual-configuration in that clientssl  profile  unchanged.

Reply:
I see this piece of code will add "profiles" list to vs payload before updating, so that the existing profiles of the vs will be removed, only client ssl profile will be there. Will that happen?

https://github.com/F5Networks/f5-openstack-agent/blob/9.8-stable/f5_openstack_agent/lbaasv2/drivers/bigip/listener_service.py#L220